modules/GraphStreaming/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>gephi-plugin-parent</artifactId>
+        <groupId>org.gephi</groupId>
+        <version>0.9.0</version>
+    </parent>
+
+    <groupId>org.gephi</groupId>
+    <artifactId>graphstreaming</artifactId>
+    <version>1.0.0</version>
+    <packaging>nbm</packaging>
+
+    <name>GraphStreaming</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.gephi.streaming</groupId>
+            <artifactId>api</artifactId>
+            <version>1.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.gephi.streaming</groupId>
+            <artifactId>impl</artifactId>
+            <version>1.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.gephi.streaming</groupId>
+            <artifactId>server</artifactId>
+            <version>1.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.gephi.desktop</groupId>
+            <artifactId>streaming</artifactId>
+            <version>1.0.0</version>
+        </dependency>
+         <dependency>
+            <groupId>org.gephi.lib</groupId>
+            <artifactId>jetty</artifactId>
+            <version>1.0.0</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>nbm-maven-plugin</artifactId>
+                <configuration>
+                    <author>Andre Panisson</author>
+                    <sourceCodeUrl>$sourcecode_url</sourceCodeUrl>
```

Comment:
Please set source code url, for instance https://github.com/panisson/gephi-plugins/tree/graph-streaming